SConstruct

```diff
@@ -120,6 +120,7 @@ opts.Add(BoolVariable('deprecated', "Enable deprecated features", True))
 opts.Add(BoolVariable('gdscript', "Enable GDScript support", True))
 opts.Add(BoolVariable('minizip', "Enable ZIP archive support using minizip", True))
 opts.Add(BoolVariable('xaudio2', "Enable the XAudio2 audio driver", False))
+opts.Add(EnumVariable('renderer', "Select Renderer", 'vulkan', ('vulkan', 'gles2', 'gles3', 'none')))
```

Comment:
GLES 3 is going to be removed immediately when Vulkan is added, it's best to remove it and then this PR will just wait until the Vulkan branch is merged and GLES 3 is removed.

Reply:
Done